### ISSUE TITLE
CSS organization

### ISSUE THREAD

**Issue by andrealeone** [LionAndTheFellows/TelegramClient#9]
Honestly: CSS is a mess at this point. My plan is to recreate the style structure in .stylus - opinions @LionAndTheFellows/main-development ?

**Comment by jackstonebridge** [comment]
Yeah! It's time to re-design the UI! Need a hand?

**Comment by andrealeone** [comment]
Of course @jackstonebridge! New branch created from 'dev-open': [_Re-styl](https://github.com/LionAndTheFellows/TelegramClient/tree/_Re-styl) - it is the branch to solve this bug. For this, 2 reviews needed. [Fetch&Pull and then PR]

**Comment by andrealeone** [comment]
@jackstonebridge, do you take charge of the PR?

**Comment by jackstonebridge** [comment]
Yep! I'll wait your approval as reviewer once we made all the stuff before *squash & merge*.
